content/en/docs/how-tos/ai-job-scheduling-kueue-integration.md

```diff
@@ -0,0 +1,773 @@
+localqueues.kueue.x-k8s.io                         2025-10-17T17:50:33Z
+resourceflavors.kueue.x-k8s.io                     2025-10-17T17:50:35Z
+```
+
+## Step-by-Step Guide
+
+### 1. Create Resource Flavor on Hub
+
+First, create a ResourceFlavor that defines the available resource types:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1 
+kind: ResourceFlavor  
+metadata:  
+  name: default-flavor
+spec:
+  # This ResourceFlavor will be used for all the resources
+```
+
+A ResourceFlavor is an object that represents the variations in the nodes available in your cluster by associating them with node labels and taints. 
+
+
+### 2. Configure ClusterQueue on Hub
+
+Set up a ClusterQueue for the workloads:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: cluster-queue
+spec:
+  cohort: ai-training
+  namespaceSelector: {} # Available to all namespaces  
+  queueingStrategy: BestEffortFIFO # Default queueing strategy 
+  resourceGroups:
+  - coveredResources: ["cpu", "memory"]
+    flavors:
+    - name: default-flavor
+      resources:
+      - name: cpu
+        nominalQuota: "32"
+      - name: memory
+        nominalQuota: "128Gi"
+```
+> NOTE: The resources need to be listed in the same order as the coveredResources list.
+
+A ClusterQueue is a cluster-scoped object that oversees a pool of resources, including CPU, memory, and GPU. It manages ResourceFlavors, sets usage limits, and controls the order in which workloads are admitted. This will handle the ResourceFlavor that was previously created.
+
+
+### 3. Create Training Namespace on Hub
+
+Create the namespace where your AI jobs will be scheduled:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: training
+  labels:
+    app: ai-training
+```
+The LocalQueue and Job require a namespace to be specified. 
+
+
+### 4. Use ClusterResourcePlacement to Propagate Cluster-Scoped Resources
+
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1beta1  
+kind: ClusterResourcePlacement  
+metadata:  
+  name: sample-kueue 
+spec:  
+  resourceSelectors:
+    - group: kueue.x-k8s.io  
+      version: v1beta1  
+      kind: ResourceFlavor  
+      name: default-flavor  
+    - group: kueue.x-k8s.io  
+      version: v1beta1  
+      kind: ClusterQueue  
+      name: cluster-queue  
+    - group: ""  
+      kind: Namespace  
+      version: v1  
+      name: training  
+      selectionScope: NamespaceOnly
+  policy:  
+    placementType: PickN  
+    numberOfClusters: 1  
+    affinity:  
+      clusterAffinity:  
+        preferredDuringSchedulingIgnoredDuringExecution:  
+          - weight: 20  
+            preference:  
+              propertySorter:  
+                name: kubernetes-fleet.io/node-count  
+                sortOrder: Descending  
+          - weight: 30  
+            preference:  
+              propertySorter:  
+                name: resources.kubernetes-fleet.io/available-cpu  
+                sortOrder: Descending  
```

Comment:
I wonder why we need to pick which clusters to put those cluster resources?

Reply:
if the namespace is on only 1 cluster, there is no other places the AI job can go